cypress/integration/Test_Phonebook/test_register.js

```diff
@@ -0,0 +1,17 @@
+describe('Register Test', function(){
+    it('Enter Details for Signup', function () {
+        // expect(true).to.equal(true);
+        cy.visit('http://localhost:3006/register')
+
+        cy.get('input#email').type('ali_1@gmail.com');
+        cy.get('input#password').type('ali123');
+        cy.get('input#username').type('ali123');
+        cy.get('input#first_name').type('ali');
+        cy.get('input#last_name').type('raza');
+        cy.get('button[type="submit"]').click().then(response=>{
+            expect(true).to.be.true
```

Comment:
Seriously? Add some real assertions please. Test whether the response is good. the status code. the token. whether the token is being stored in the localStorage etc.